requirements/prod.txt

```diff
@@ -1,12 +1,12 @@
-Django==1.7.7
-Pillow==2.9.0
-django-allauth==0.19.1
-django-braces==1.8.0
-django-cities-light==3.1.2
-django-ckeditor==4.4.8
-django-crispy-forms==1.4.0
-django-guardian==1.2.5
-django-imagekit==3.2.6
-djangocms-admin-style==0.2.5
-psycopg2==2.6
+Django
```

Comment:
For everything please specify the version. Without version the latest will be pulled out in Travis and elsewhere which makes things a bit unstable.

Reply:
Hey @bhavya-reddy !
Are you still working on it?